CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
 # dbt_reddit_ads_source v0.1.0
 
 ## Initial Release
-- This is the initial release of this package. For more information refer to the [README](/README.md).
+- This is the initial release of this package. For more information refer to the [README](/README.md). 
```

Comment:
Can you change this to be the "What does this package do" from the README. Just to add a bit more context to the release notes.

Reply:
updated

Reply:
@fivetran-reneeli this looks like it wasn't updated. Would you be able to update to be the same wording as the README section.

Reply:
oh my bad, totally misread and thought to just update the link to the readme to that specific section. Ok, I added in the notes from the readme